docs/source/algorithms/MSDFit-v1.rst

```diff
@@ -30,22 +33,36 @@ Usage
     # Create some data that is similar to the output of ElasticWindowMultiple
     sample = CreateSampleWorkspace(Function='User Defined',
                     UserDefinedFunction='name=ExpDecay,Height=1,Lifetime=6',
-                    NumBanks=1, BankPixelWidth=1, XUnit='QSquared', XMin=0.0,
+                    NumBanks=1, BankPixelWidth=1, XUnit='momentum', XMin=0.0,
                     XMax=5.0, BinWidth=0.1)
 
-    msd, param, fit = MSDFit(InputWorkspace=sample,
-                             XStart=0.0, XEnd=5.0,
-                             SpecMin=0, SpecMax=0)
+    g_msd, g_param, g_fit = MSDFit(InputWorkspace=sample,
+                                   Model="Gauss",
+                                   XStart=0.0, XEnd=5.0,
+                                   SpecMin=0, SpecMax=0)
+
+    y_msd, y_param, y_fit = MSDFit(InputWorkspace=sample,
+                                   Model="Yi",
+                                   XStart=0.0, XEnd=5.0,
+                                   SpecMin=0, SpecMax=0)
 
-    print 'A0: ' + str(msd.readY(0))
-    print 'A1: ' + str(msd.readY(1))
+    print 'Using Gauss Model'
+    print 'A0: ' + str(g_msd.readY(0))
+    print 'A1: ' + str(g_msd.readY(1))
+    print 'Using Yi Model'
+    print 'A0: ' + str(y_msd.readY(0))
+    print 'A1: ' + str(y_msd.readY(1))
```

Comment:
Please use the print function: `print(...)`. We need to convert all the usage examples to Python 3 at some point, anyway.

Reply:
or logger.information

Reply:
Logger output won't be checked by the doctests, so I would still recommend using `print()`.